packages/core/src/tracing/instrumentation/cloud/aws-sdk/v2/lambda.js

```diff
@@ -39,7 +38,7 @@ class InstanaAWSLambda extends InstanaAWSProduct {
 
       const instanaCustomHeaders = {
         Custom: {
-          'x-instana-l': cls.tracingSuppressed() ? '0' : '1',
+          'x-instana-l': '1',
```

Comment:
TBD when PR is ready to review

params never used, looks like a bug
do we need to forward headers for lambda invoke on suppression